abi/encoding_test.go

```diff
@@ -335,6 +335,49 @@ func TestEncoding(t *testing.T) {
 	}
 }
 
+func TestEncoding2(t *testing.T) {
+	cases := []struct {
+		Type  string
+		Input interface{}
+	}{
+		{
+			"tuple(bytes a)[]",
+			[]map[string]interface{}{
+				{
+					"a": []byte{
+						0xff,
+						0xff,
+					},
+				},
+				{
+					"a": []byte{
+						0xff,
+						0xff,
+					},
+				},
+			},
+		},
+	}
+
+	server := testutil.NewTestServer(t)
+
+	for _, c := range cases {
+		c := c
+		t.Run(c.Type, func(t *testing.T) {
+			t.Parallel()
+
+			tt, err := NewType(c.Type)
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			if err := testEncodeDecode(t, server, tt, c.Input); err != nil {
+				t.Fatal(err)
+			}
+		})
+	}
+}
```

Comment:
Is this pushed by mistake? Can we make it as a part of some existing unit test as a sub test case.